### DISCUSSION TITLE
Can PandaCSS show me the names of the styled components in the browser inspector?

### DISCUSSION THREAD

**Discussion by leodevbro** [chakra-ui/panda#1297] [Q&A]
When I inspect some div boxes in the browser, I find it difficult to find the specific div boxes in the source code (to edit it), because there is not any reference names in class names. There are only styling descriptions.

![image](https://github.com/chakra-ui/panda/assets/54504880/31edd975-5591-45c1-8cd7-00ed26952768)

The source code is this:

![image](https://github.com/chakra-ui/panda/assets/54504880/898b7ce8-3f97-4e1a-8245-6aaf9bf05978)

Yes, this is a very simple example, sure I can find it in the source code, but it is a problem for big projects.

**So, can PandaCSS generate class names like this?**

![image](https://github.com/chakra-ui/panda/assets/54504880/82d72ed1-e8ff-4448-b175-81d015d09de2)
to be able to search `MyNiceDiv` in VS Code.

**Comment by anubra266** [comment]
@leodevbro No panda can't do this. And I'm not sure I understand this. Would be helpful if you have a precedent. Like how some other css engines do this.

**Reply by leodevbro** [reply]
`styled-components` does this in my project:

![image](https://github.com/chakra-ui/panda/assets/54504880/de638d7c-c703-4c8e-81e7-876906ce9f6d)

**Reply by anubra266** [reply]
@leodevbro I'm referring to atomic solutions like Tailwind. Because Panda is atomic.

**Reply by leodevbro** [reply]
OK, so, I'm wondering, how do you find a div in source code from browser inspect view?

**Reply by anubra266** [reply]
@leodevbro I right click on the div in the browser, and inspect tools. It takes me to the code for that div

**Reply by leodevbro** [reply]
inspect? You mean this inspect button?
![image](https://github.com/chakra-ui/panda/assets/54504880/62fed284-50dd-4d1f-a519-0e4dffdf920a)

It's not what I am saying. This inspect button will just show you the div in the browser developer window, not in the actual codebase (In VS Code). I want to somehow find the div in the VS Code codebase (Actual project files, I mean ReactJS project files).

**Reply by anubra266** [reply]
@leodevbro That's outside the scope of panda.
